scripts/test-react.js

```diff
@@ -81,6 +81,7 @@ function runTestSuite(outputJsx, shouldTranspileSource) {
     reactOutput: outputJsx ? "jsx" : "create-element",
     inlineExpressions: true,
     invariantLevel: 0,
+    simpleClosures: true,
```

Comment:
This will unlock a whole new set of bugs.

Reply:
We *do* use it for `debug-fb-www` script though. So this moves us closer to how we currently generate the bundles.

Does it mean it's too unstable and we shouldn't?

Reply:
It is completely broken. Let's remove it and fix the normal stuff instead. E.g. by optimizing it if we need to. We need to do something because it'll definitely generate wrong code in current scenarios.